requirements-dev.in

```diff
@@ -3,7 +3,6 @@
 pip-tools>=5.4.0
 
 coverage==4.5.3
```

Comment:
We can probably get rid of coverage altogether

Reply:
removed in 919a4c3